client/state/data-layer/wpcom/sites/scan/index.js

```diff
@@ -48,7 +48,6 @@ const formatScanStateRawResponse = ( { state, threats, credentials, most_recent:
 		mostRecent: mostRecent
 			? {
 					...mostRecent,
-					timestamp: new Date( mostRecent.timestamp ),
```

Comment:
Had to remove this as by default, `Date` assumes a TZ of the browser. If we want to keep this, we'll need to specify UTC in `timestamp` or switch to a true Unix timestamp.